testing.sagebrush.master/Stepwat_Inputs/Input/sxw/Input/siteparam.in

```diff
@@ -1,75 +1,99 @@
+
+#---- Evaporation
 # These control the tangent function (tanfunc) which affects the amount of soil
-# water extractable by evaporation and transpiration.
-# These constants aren't documented by the ELM doc.
-45.			# rate shift (x value of inflection point).  lower value shifts curve
-			# leftward, meaning less water lost to evap at a given swp.  effectively
-			# shortens/extends high rate.
-.1			# rate slope: lower value (eg .01) straightens S shape meaning more gradual
-			# reduction effect; higher value (.5) makes abrupt transition
-.25			# inflection point (y-value of inflection point)
-0.5			# range: diff btw upper and lower rates at the limits
-
-# ---- Transpiration Coefficients ----
-# comments from Evap constants apply.
-45.          # rate shift
-.1          # rate shape
-.5          # inflection point
-1.1          # range
-
-# ---- Soil Temperature Constants ----
+# water extractable by bare-soil evaporation and transpiration.
+45.		# rate shift (x value of inflection point).  lower value shifts curve
+		# leftward, meaning less water lost to evap at a given swp.  effectively
+		# shortens/extends high rate.
+.1		# rate slope: lower value (eg .01) straightens S shape meaning more gradual
+		# reduction effect; higher value (.5) makes abrupt transition
+.25		# inflection point (y-value of inflection point)
+0.5		# range: diff btw upper and lower rates at the limits
+
+#---- Transpiration
+# These control the tangent function (tanfunc) which affects the amount of soil
+# water extractable by transpiration.
+45.		# rate shift
+.1		# rate shape
+.5		# inflection point
+1.1		# range
+
+#---- Surface and soil temperature
 # from Parton 1978, ch. 2.2.2 Temperature-profile Submodel
-300.			# biomass limiter, 300 g/m^2 in Parton's equation for T1(avg daily temperature at the top of the soil)
-15.			# constant for T1 equation (used if biomass <= biomass limiter), 15 in Parton's equation
--4.			# constant for T1 equation (used if biomass > biomass limiter), -4 in Parton's equation
-600.        # constant for T1 equation (used if biomass > biomass limiter), 600 in Parton's equation
+300.		# biomass limiter, 300 g/m^2 in Parton's equation for T1(avg daily temperature at the top of the soil)
+15.		# constant for T1 equation (used if biomass <= biomass limiter), 15 in Parton's equation
+-4.		# constant for T1 equation (used if biomass > biomass limiter), -4 in Parton's equation
+600.		# constant for T1 equation (used if biomass > biomass limiter), 600 in Parton's equation
 0.00070		# constant for cs (soil-thermal conductivity) equation, 0.00070 in Parton's equation
-0.00030     # constant for cs equation, 0.00030 in Parton's equation
+0.00030		# constant for cs equation, 0.00030 in Parton's equation
 0.18		# constant for sh (specific heat capacity) equation, 0.18 in Parton's equation
-6.69		# constant mean air temperature (the soil temperature at the lower boundary, 180 cm) in celsius
```

Comment:
Why was the default value for "constant mean air temperature" changed from 6.69 to 4.15? Check with Kyle and Martin how this affects the wrapper or needs to be reverted